ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -167,24 +167,25 @@ fi
 
 if ver hwcmp MINDPX_V2
 then
-	if mpu6500 start
+	# External I2C bus
+	if hmc5883 -C -T -X start
 	then
 	fi
 
-	if lsm303d start
+	# Internal I2C bus
+	if hmc5883 -C -T -I -R 8 start
 	then
 	fi
 
-	if l3gd20 start
+	if mpu6500 -R 8 start
 	then
 	fi
 
-	# External I2C bus
-	if hmc5883 -C -T -X start
+	if lsm303d -R 10 start
 	then
 	fi
 
-	if lis3mdl -R 2 start
+	if l3gd20 -R 14 start
```

Comment:
@eflyinghi Use command option to do sensor rotation.

[Code here is ugly and none-maintainability, especially when there are more than one sensors with same type but different coordinate axis in the system. ](https://github.com/airmind/OpenMindPX/pull/1/commits/717ce1530bb0f573d7e3aab3483804518b06e071#diff-fc165566877bb4af47d4be96eb8b7690L1580)